test_util/launch.py

```diff
@@ -1,20 +1,282 @@
-"""
+"""DC/OS Launch
 
 Usage:
-dcos-launch create [--wait] [--dump-info=cluster_info.json] <config=config.yaml>
-dcos-launch wait <path=cluster_info.json>
-dcos-launch describe <path=cluster_info.json>
-dcos-launch delete <path=cluster_info.json>
+  dcos-launch create [--config-path=<path>] [--info-path=<path>]
+  dcos-launch wait [--info-path=<path>]
+  dcos-launch describe [--info-path=<path>]
+  dcos-launch test [--info-path=<path>] [--] [<test_command>]...
```

Comment:
This does have the capability to run any arbitrary test (when the user provides the specific arbitrary command) so the name run-internal-integration-tests is also a little bit misleading